### PR TITLE
Tup#222 Introduce header new format

### LINKED ISSUE #222: Re-design menu
To improve UI/UX, i propose to do following.

- Hide navigation menu
- Move home botton to the side of message botton on the upper right.
- Move space selection drop down from the uppor right to the upper left, and show there space icon/name when user is in a space.
- Move space menu from the side and put it in a drop down button on the uper left.

(All the items Spaces, People, Project and Marketplace) will be added later under search buttom, after alignment with Malek how to best design them)

grafik

### PR DESCRIPTION
**Pull request by craxrev**
Fixes Coinsence/meta#222

### CODE REVIEW

File: themes/Coinsence/views/layouts/main.php [hunk 1]
```diff
@@ -44,31 +54,22 @@
...
-
                 <div class="search pull-right hidden-xs">
                     <?= \yii\helpers\Html::a(
                         '<i class="fa fa-search"></i>',
                         ['/search']) ?>
                 </div>
 
+                <div class="home pull-right hidden-xs">
+                    <?= \yii\helpers\Html::a(
+                        '<i class="fa fa-home"></i>',
```
Comment (gdalyy) [inline themes/Coinsence/views/layouts/main.php]: why multiple lines here ?

File: themes/Coinsence/views/space/space/_layout.php [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+/**
+ * @var \humhub\modules\ui\view\components\View $this
+ * @var \humhub\modules\space\models\Space $space
+ * @var string $content
+ */
+
+/** @var \humhub\modules\content\components\ContentContainerController $context */
+$context = $this->context;
+$space = $context->contentContainer;
```
Comment (gdalyy) [inline themes/Coinsence/views/space/space/_layout.php:16]: add var annotations here too

File: themes/Coinsence/views/space/widgets/spaceChooser.php [hunk 3]
```diff
@@ -95,4 +96,24 @@
         </li>
 
     </ul>
+    <script>
+        if (humhub.modules.event) {
+            const event = humhub.modules.event;
+            event.on('humhub:modules:space:chooser:afterInit', function(evt, spaceChooser) {
+                console.log('humhub:modules:space:chooser:afterInit');
```
Comment (gdalyy) [inline themes/Coinsence/views/space/widgets/spaceChooser.php:103]: I think we should remove console logs !

**Review by gdalyy — CHANGES_REQUESTED** [review]

**Comment by gdalyy** [comment]
You have a typo in PR name :yum:

File: themes/Coinsence/views/space/space/_layout.php (hunk 2 above)
Comment (craxrev) [reply themes/Coinsence/views/space/space/_layout.php:16]: this is their default template, also there are some var annotation above

**Review by gdalyy — APPROVED** [review]

**Pull request merged**